dodo.py

```diff
@@ -285,7 +325,7 @@ def common_branch(root):
 def common_push(root):
     return {
         'actions': [
-            'cd %s/template && git checkout update && git push -u origin update' % root,
+            'cd %s/template && git checkout -B update && git push -u origin update' % root,
```

Comment:
What's the reason for this change?

Reply:
It was because the Flask repo did not have a branch called `update`. Likely unnecessary now that the branch does exist, right? I can remove it.

Reply:
The branch should be created in the `common_branch()` step.  The push step assumes there's already an update branch to be pushed; if there isn't then adding the `-B` here will make an empty PR, as the new branch will be exactly the same as master.

At least that's how I think it should all work. :smile: